sys/win32/midi-win32mm.c

```diff
@@ -58,13 +58,12 @@ static unsigned int last_known_out_port = 0;
 static void _win32mm_sysex(LPMIDIHDR *q, const unsigned char *d, unsigned int len)
 {
 	char *z;
-	LPMIDIHDR m;
+	LPMIDIHDR m = {};
```

Comment:
Looks like AppVeyor doesn't like this change:

```
sys/win32/midi-win32mm.c: In function '_win32mm_sysex':
sys/win32/midi-win32mm.c:61:16: error: empty scalar initializer
  LPMIDIHDR m = {};
                ^
sys/win32/midi-win32mm.c:61:16: note: (near initialization for 'm')
make[1]: *** [Makefile:1326: sys/win32/midi-win32mm.o] Error 1
make[1]: Leaving directory '/c/projects/schismtracker'
```

Reply:
force pushed fix, thanks